### PR TITLE
remove vestiges of old updater

### PR DESCRIPTION
**Pull request by afragen**
Here it is if you're ready to break away.

### CODE REVIEW

File: init.php [hunk 1]
```diff
@@ -39,19 +39,11 @@
 }
 else {
     // Current version
-    define( 'PODS_VERSION', '2.5.2-b-1' );
+    define( 'PODS_VERSION', '2.5.2.1' );
```
Comment (sc0ttkclark) [inline init.php]: Remove the version changes, we're releasing a new release in the next week.

File: init.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Plugin Name: Pods - Custom Content Types and Fields
 Plugin URI: http://pods.io/
 Description: Pods is a framework for creating, managing, and deploying customized content types and fields
-Version: 2.5.2 Beta 1
+Version: 2.5.2.1
```
Comment (sc0ttkclark) [inline init.php]: Remove the version changes, we're releasing a new release in the next week.

**Comment by sc0ttkclark** [comment]
Needs a `release/3.0` version of this PR

**Comment by afragen** [comment]
I'll revert the version changes and submit similar PR to `release/3.0` tomorrow.

**Pull request merged**